.github/workflows/ci.yml

```
@@ -0,0 +1,143 @@
+      fail-fast: false
+      matrix:
+        runs-on: [ubuntu-latest]
+        sys:
+        - {compiler: gcc, version: '9', name: tbb, flags: -DXTENSOR_USE_TBB=ON -DTBB_INCLUDE_DIR=$CONDA_PREFIX/include -DTBB_LIBRARY=$CONDA_PREFIX/lib}
+        - {compiler: gcc, version: '10', name: openmp, flags: -DXTENSOR_USE_OPENMP=ON}
+        - {compiler: gcc, version: '11', name: noexcept, flags: -DXTENSOR_DISABLE_EXCEPTION=ON}
+        - {compiler: gcc, version: '11', name: xsimd, flags: -DXTENSOR_USE_XSIMD=ON}
+        - {compiler: gcc, version: '11', name: c++17, flags: -DCPP17=ON}
+        - {compiler: clang, version: '15', name: assert, flags: -DXTENSOR_ENABLE_ASSERT=ON}
+        - {compiler: clang, version: '16', name: column-major, flags: -DDEFAULT_COLUMN_MAJOR=ON}
+        include:
+        - config: -Bbuild -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX -DBUILD_TESTS=ON
+
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    name: ${{ matrix.runs-on }} • ${{ matrix.sys.compiler }} ${{ matrix.sys.version }} - ${{ matrix.sys.name }}
+    runs-on: ${{ matrix.runs-on }}
+    steps:
+
+    - name: Setup GCC
+      if: ${{ matrix.sys.compiler == 'gcc' }}
+      run: |
+        GCC_VERSION=${{ matrix.sys.version }}
+        CC=gcc-$GCC_VERSION
+        echo "CC=$CC" >> $GITHUB_ENV
+        CXX=g++-$GCC_VERSION
+        echo "CXX=$CXX" >> $GITHUB_ENV
+
+    - name: Setup clang
+      if: ${{ matrix.sys.compiler == 'clang' }}
+      run: |
+        LLVM_VERSION=${{ matrix.sys.version }}
+        #sudo add-apt-repository ppa:ubuntu-toolchain-r/test || exit 1
+        wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add - || exit 1
+        if [[ $LLVM_VERSION -ge 13 ]]; then
+          sudo add-apt-repository "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-$LLVM_VERSION main" || exit 1
+        else
+          sudo add-apt-repository "deb http://apt.llvm.org/focal/ llvm-toolchain-focal main" || exit 1
+        fi || exit 1
+        sudo apt-get update || exit 1
+        sudo apt-get --no-install-suggests --no-install-recommends install clang-$LLVM_VERSION || exit 1
+        sudo apt-get --no-install-suggests --no-install-recommends install g++-9 g++-9-multilib || exit 1
+        sudo ln -s /usr/include/asm-generic /usr/include/asm
+        CC=clang-$LLVM_VERSION
+        echo "CC=$CC" >> $GITHUB_ENV
+        CXX=clang++-$LLVM_VERSION
+        echo "CXX=$CXX" >> $GITHUB_ENV
+
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set conda environment
+      if: ${{ matrix.sys.name != 'tbb' }}
+      uses: mamba-org/provision-with-micromamba@main
+      with:
+        environment-name: myenv
+        environment-file: environment-dev.yml
+
+    - name: Set conda environment
+      if: ${{ matrix.sys.name == 'tbb' }}
+      uses: mamba-org/provision-with-micromamba@main
+      with:
+        environment-name: myenv
+        environment-file: environment-dev.yml
+        extra-specs: |
+          tbb-devel
```

Is there a way do something like
```suggestion
    - name: Set conda environment
      uses: mamba-org/provision-with-micromamba@main
      with:
        environment-name: myenv
        environment-file: environment-dev.yml
        extra-specs: ${{ matrix.sys.extra }}
```
where `${{ matrix.sys.extra }}` could be an (empty) list. @wolfv you might know